selfdrive/manager/process_config.py

```diff
@@ -28,7 +28,6 @@
   PythonProcess("paramsd", "selfdrive.locationd.paramsd"),
   PythonProcess("plannerd", "selfdrive.controls.plannerd"),
   PythonProcess("radard", "selfdrive.controls.radard"),
-  PythonProcess("rtshield", "selfdrive.rtshield", enabled=EON),
```

Comment:
Also delete rtshield?

Reply:
will keep around in selfdrive/debug. it's pretty handy for testing